dev-packages/localization-manager/package.json

```diff
@@ -33,7 +33,7 @@
     "deepmerge": "^4.2.2",
     "fs-extra": "^4.0.2",
     "glob": "^7.2.0",
-    "typescript": "^4.4.3"
+    "typescript": "~4.4.4"
```

Comment:
Minor: Please update this to `~4.5.5` as well.

Reply:
Good catch.